package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "node app.js",
     "dev": "nodemon ./app.js",
-    "test": "eslint ./**.js"
+    "test": "mocha --reporter spec"
```

Comment:
This does not belong here. Neither does the `package-lock.json` file.

Reply:
you mean the file itself ?

Reply:
Yea, the changes to it should probably be reverted. You probably installed some packages for testing mocha and didn't uninstall them correctly, so they're still in the lock file. But it's not the end of the world so I'll approve it anyway.